luigi_pipeline/seqr_gcnv_loading.py

```diff
@@ -25,6 +76,16 @@ class SeqrGCNVVariantMTTask(SeqrVCFToVariantMTTask):
     RUN_VEP = False
     SCHEMA_CLASS = SeqrGCNVVariantSchema
 
+    is_new_joint_call = luigi.BoolParameter(default=False, description='Is this a fully joint-called callset.')
+
+    @staticmethod
+    def validate_ht_types(ht, expected_types=EXPECTED_TYPES):
```

Comment:
This throws if there are extra columns or incorrectly typed columns but not if there are missing columns.  If those arise, the pipeline will fail anyway.  Might be better to be explicit here though?